src/libsyntax/feature_gate.rs

```diff
@@ -199,9 +199,6 @@ declare_features! (
 
     // no-tracking-issue-end
 
-    // Allows using `#[unsafe_destructor_blind_to_params]` (RFC 1238).
-    (active, dropck_parametricity, "1.3.0", Some(28498), None),
```

Comment:
This should be moved to the removed features section lower down in this file

Reply:
done.